msteams-platform/resources/dev-preview/developer-preview-intro.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Public developer preview for Microsoft Teams
-description: In this article, know the features that are in the Public Developer Preview of Microsoft Teams and developer preview app manifest.
+description: A Developer Preview is public program for developers to explore and test upcoming features for potential inclusion in your Microsoft Teams app.
```

Comment:
@surbhigupta12  - Most of the platform use Beta term for this. Can we include this in description so that developers land on this page?